src/api/api.tsx

```diff
@@ -0,0 +1,112 @@
+import axios from "axios";
+
+type Filter = {
+  limit?: number;
```

Comment:
type QueryParams = Record<string, string>;

const queryParams: QueryParams = {
  param1: "value1",
  param2: "value2"
};

this type describe query params very well

Reply:
done